app/models/user.rb

```diff
@@ -1,3 +1,9 @@
 class User < ApplicationRecord
-  has_many :posts, :likes, :comments
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ]  According to the diagram, you should add the foreign key `author_id` to posts, comments, and likes:
```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```